seaborn/tests/test_categorical.py

```diff
@@ -1532,6 +1594,167 @@ def test_three_strip_points(self):
         nt.assert_equal(ax.collections[0].get_facecolor().shape, (1, 4))
 
 
+class TestSwarmPlotter(CategoricalFixture):
+
+    default_kws = dict(x=None, y=None, hue=None, data=None,
+                       order=None, hue_order=None, split=False,
+                       orient=None, color=None, palette=None)
+
+    def test_overlap(self):
+
+        p = cat._SwarmPlotter(**self.default_kws)
+        nt.assert_false(p.overlap((0, 0), (1, 1), np.sqrt(2)))
```

Comment:
Test failing because sqrt(2)**2 = 2.0000004 ... you can use (0,0), (3,4) which should make an exact distance of 5 betwwen the 2 circles center...